htdocs/salaries/card.php

```diff
@@ -496,7 +496,7 @@
 // Create
 if ($action == 'create' && $permissiontoadd) {
 	$year_current = dol_print_date(dol_now('gmt'), "%Y", 'gmt');
```

Comment:
I see that there is a mixture here: 
- $year_current was already using the 'gmt' option, but,
- $pastmonth was using strftime.

Is the correct approach to add $gmt here or not?